plugins/medianpoc/plugin.go

```diff
@@ -103,12 +111,39 @@ func (p *Plugin) newFactory(ctx context.Context, config core.ReportingPluginServ
 		spec:           jfp,
 		lggr:           p.Logger,
 	}
+
+	var gds median.DataSource
+	gp, err := jc.getPipeline("gasPriceSubunitsPipeline")
+
+	_, pipelineNotFound := err.(*PipelineNotFoundError)
+	if !pipelineNotFound && err != nil {
+		return nil, err
+	}
+
+	// Setting includeGasPriceSubunitsInObservation properly ensures we do not break libocr codec
```

Comment:
we had talked about updating this comment to be a more direct and simpler -- we have to omit the field in to maintain backward compatibility